run/order.go

```diff
@@ -145,22 +146,50 @@ func BuildDAG(
 
 	visited[s.Path()] = struct{}{}
 
-	logger.Trace().
-		Msg("Load all stacks in dir after current stack.")
-	afterStacks, err := loader.LoadAll(root, s.HostPath(), s.After()...)
+	cleanOrderPaths := func(fieldname string, paths []string) []string {
```

Comment:
```suggestion
	orderAbsPaths := func(fieldname string, paths []string) []string {
```

Or:

```suggestion
	normalizePaths := func(fieldname string, paths []string) []string {
```

Since you are normalizing all paths (rel/abs to project) to paths absolute to the host. Feel free to disregard.

Reply:
it's actually not normalizing neither making them absolute.. but just filtering out the ones with error (no such file, not directory, no permission, etc).